services/npm/npm.js

```diff
@@ -1,54 +1,12 @@
 'use strict';
 
-const { BaseJsonService } = require('../base');
-const {
-  makePackageDataUrl,
-  typeDefinitions,
-} = require('../../lib/npm-badge-helpers');
-
-module.exports = class NPMTypeDefinitions extends BaseJsonService {
-  static get category() {
-    return 'version';
-  }
-
-  static get defaultBadgeData() {
-    return { label: 'type definitions' };
-  }
-
-  static get url() {
-    return {
-      base: 'npm/types',
-      format: '(?:@([^/]+)/)?([^/]+)',
-      capture: ['scope', 'packageName'],
-      queryParams: ['registry_uri'],
-    };
-  }
-
-  static get examples() {
-    return [{
-      title: 'npm type definitions',
-      previewUrl: 'chalk',
-      keywords: ['node', 'typescript', 'flow'],
-    }];
-  }
-
-  async handle({ scope, packageName }, { registry_uri: registryUrl }) {
-    const apiUrl = makePackageDataUrl({ registryUrl, scope, packageName });
-
-    const json = await this._requestJson(apiUrl, {}, 'package not found');
-
-    let packageData;
-    if (scope === undefined) {
-      packageData = json;
-    } else {
-      const latestVersion = json['dist-tags'].latest;
-      packageData = json.versions[latestVersion];
-    }
-
-    const message = typeDefinitions(packageData);
-    return {
-      message,
-      color: message === 'none' ? 'lightgray' : 'blue',
-    };
-  }
-};
+const NpmVersion = require('./npm-version');
+const NpmLicense = require('./npm-license');
+const NpmDownloads = require('./npm-downloads');
+const NpmTypeDefinitions = require('./npm-type-definitions');
+
+module.exports = [
+  NpmVersion,
+  NpmLicense,
+  NpmTypeDefinitions,
+].concat(NpmDownloads);
```

Comment:
I think this can now be removed in favour of renaming the files as per #1797.
Similarly `/node/node.js` wants to be `/node/node.service.js`.